tests/suites/test_suite_psa_crypto_slot_management.data

```diff
@@ -1,3 +1,21 @@
+Check key ID range: MBEDTLS_PSA_KEY_ID_BUILTIN_MIN
+check_key_id_value: MBEDTLS_PSA_KEY_ID_BUILTIN_MIN
+
+Check key ID range: MBEDTLS_PSA_KEY_ID_BUILTIN_MAX
+check_key_id_value: MBEDTLS_PSA_KEY_ID_BUILTIN_MAX
+
+Check key ID range: PSA_KEY_ID_USER_MIN
+check_key_id_value: PSA_KEY_ID_USER_MIN
+
+Check key ID range: PSA_KEY_ID_USER_MAX
+check_key_id_value: PSA_KEY_ID_USER_MAX
+
+Check key ID range: PSA_KEY_ID_VOLATILE_MIN
+check_key_id_value: PSA_KEY_ID_VOLATILE_MIN
+
+Check key ID range: PSA_KEY_ID_VOLATILE_MAX
+check_key_id_value: PSA_KEY_ID_VOLATILE_MAX
+
```

Comment:
Please add a test case with a key ID is not in the builtin range, not in the volatile range and not in the user range. (I think `PSA_KEY_ID_VOLATILE_MAX` works, but the ranges are a bit complicated, we should make sure that whatever value we pick, the compilation or the test case fails if the value turns out to be in one of the known ranges after all.)